src/main/js/main_html5.js

```diff
@@ -1483,8 +1483,21 @@ require("../../../html5-common/js/utils/environment.js");
         this.controller.notify(this.controller.EVENTS.PAUSED);
       }
       forceEndOnPausedIfRequired();
+      removeControlsAttr();
```

Comment:
@KseniiaSokolovaEpam I think it should be possible to listen to the `webkitendfullscreen` event on the video element so that the fix is executed exactly when fullscreen mode is exited. If not, then this is approach should be good as well. Good catch btw